### PR TITLE
feat: Navigation Bar added

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**

### CODE REVIEW

**Comment by SuyashTiwariCoder** [comment]
nav

**Review by ajit-sharmaGH — APPROVED** [review]
Looks fine

File: Utilities/Text Utility/textUtil.html [hunk 1]
```diff
@@ -90,75 +91,78 @@ <h5 class="h5 m2">Headline 5 </h5>
...
 &lt;p class="txt--sm">This is a small Text . &lt;/p></code>
   </pre>
-        <button id="copy-btn">Copy</button>
-    </div>
+                        <button id="copy-btn">Copy</button>
+                    </div>
 
 
-    <div class="divider"></div>
+                    <div class="divider"></div>
```
Comment (Kothalkarpriya) [inline Utilities/Text Utility/textUtil.html:115]: remove extra blank spaces please

**Review by Kothalkarpriya — CHANGES_REQUESTED** [review]
Do use code formatter and there are lots of spaces in your code please remove

File: Utilities/Text Utility/textUtil.html (hunk 1 above)
Comment (SuyashTiwariCoder) [reply Utilities/Text Utility/textUtil.html:115]: I have used beautify to format the code .  You can see the changed code its more readable now .

**Review by Kothalkarpriya — APPROVED** [review]

**Pull request merged**